datahub/detector/dtree/api.py

```diff
@@ -0,0 +1,14 @@
+from url_fetcher import UrlFetcher
```

Comment:
Add Apache 2.0 License into header of every Python file.
You can config your IDE for automatically do this.